client/src/main/java/io/split/client/SplitClientImpl.java

```diff
@@ -75,67 +75,67 @@ public SplitClientImpl(SplitFactory container,
+        return getTreatment(key, featureFlag, Collections.<String, Object>emptyMap());
     }
 
     @Override
-    public String getTreatment(String key, String split, Map<String, Object> attributes) {
-        return getTreatmentWithConfigInternal(key, null, split, attributes, MethodEnum.TREATMENT).treatment();
+    public String getTreatment(String key, String featureFlag, Map<String, Object> attributes) {
+        return getTreatmentWithConfigInternal(key, null, featureFlag, attributes, MethodEnum.TREATMENT).treatment();
     }
 
     @Override
-    public String getTreatment(Key key, String split, Map<String, Object> attributes) {
-        return getTreatmentWithConfigInternal(key.matchingKey(), key.bucketingKey(), split, attributes, MethodEnum.TREATMENT).treatment();
+    public String getTreatment(Key key, String featureFlag, Map<String, Object> attributes) {
+        return getTreatmentWithConfigInternal(key.matchingKey(), key.bucketingKey(), featureFlag, attributes, MethodEnum.TREATMENT).treatment();
     }
 
     @Override
-    public SplitResult getTreatmentWithConfig(String key, String split) {
-        return getTreatmentWithConfigInternal(key, null, split, Collections.<String, Object>emptyMap(), MethodEnum.TREATMENT_WITH_CONFIG);
+    public SplitResult getTreatmentWithConfig(String key, String featureFlag) {
+        return getTreatmentWithConfigInternal(key, null, featureFlag, Collections.<String, Object>emptyMap(), MethodEnum.TREATMENT_WITH_CONFIG);
     }
 
     @Override
-    public SplitResult getTreatmentWithConfig(String key, String split, Map<String, Object> attributes) {
-        return getTreatmentWithConfigInternal(key, null, split, attributes, MethodEnum.TREATMENT_WITH_CONFIG);
+    public SplitResult getTreatmentWithConfig(String key, String featureFlag, Map<String, Object> attributes) {
+        return getTreatmentWithConfigInternal(key, null, featureFlag, attributes, MethodEnum.TREATMENT_WITH_CONFIG);
     }
 
     @Override
-    public SplitResult getTreatmentWithConfig(Key key, String split, Map<String, Object> attributes) {
-        return getTreatmentWithConfigInternal(key.matchingKey(), key.bucketingKey(), split, attributes, MethodEnum.TREATMENT_WITH_CONFIG);
+    public SplitResult getTreatmentWithConfig(Key key, String featureFlag, Map<String, Object> attributes) {
+        return getTreatmentWithConfigInternal(key.matchingKey(), key.bucketingKey(), featureFlag, attributes, MethodEnum.TREATMENT_WITH_CONFIG);
     }
 
     @Override
-    public Map<String, String> getTreatments(String key, List<String> splits) {
-        return getTreatments(key, splits, Collections.emptyMap());
+    public Map<String, String> getTreatments(String key, List<String> featureFlag) {
+        return getTreatments(key, featureFlag, Collections.emptyMap());
     }
 
     @Override
-    public Map<String, String> getTreatments(String key, List<String> splits, Map<String, Object> attributes) {
```

Comment:
Should use featureFlags instead of featureFlag

Reply:
Thank you! I updated.